### PR TITLE
Add what's new for v0.7.0

### PR DESCRIPTION
**Pull request by leewujung**
This PR adds what's new section for v0.7.0

**Reminder** -- to replace DOC with URL for data processing levels documentation.

### CODE REVIEW

File: docs/source/whats-new.md [hunk 1]
```diff
@@ -4,6 +4,42 @@ What's new
...
+This release includes new features to interface with Echoview ECS files for computing Sv, reorganization of computing functions into new subpackages, addition of data processing levels to data products from a subset of computing functions, a number of additions and changes associated with metadata, and bug fixes for environmental and calibration parameters intake for computing Sv.
+
+## New features and major changes
+- Allow using ECS for calibrating Simrad echosounders (#996, #1004)
+  - This functionality is in a **beta** testing stage
+  - Details of implementation may change and bugs are possible
+  - Expand ECS parser to accept frequency-dependent values in EK80 ECS files
+- Overhaul `env_params` to ensure correct intake for calibration (#985)
+  - Now allows using `env_params` entries that are xr.DataArrays
+- Move functions previously in  `preprocess` subpackage to new subpackages (#993)
```
Comment (emiliom) [inline docs/source/whats-new.md:20]:
```suggestion
- Move functions previously in `preprocess` subpackage to new subpackages (#993). Calling these functions from `preprocess` is deprecated and will be removed in v0.7.1.
```

File: docs/source/whats-new.md [hunk 2]
```diff
@@ -4,6 +4,42 @@ What's new
...
+  - `clean`: `remove_noise`, `estimate_noise`
+  - `commongrid`: `compute_MVBS`, `compute_MVBS_index_binning`
+- Add `commongrid.compute_NASC` (#1005)
+  - The current implementation uses brute force looping for mean Sv computation, this will be refactored and optimized together with other functions requiring the same pattern in an upcoming release
+- Add global attributes for data processing levels (#1001)
+  - See **DOC** for functions and conditions under which such attributes are added
```
Comment (emiliom) [inline docs/source/whats-new.md:26]:
```suggestion
- Add global attributes for data processing levels (#1001).
  - This functionality is in a **beta** testing stage.
  - See **DOC** for functions and conditions under which such attributes are added
```

File: docs/source/whats-new.md [hunk 3]
```diff
@@ -4,6 +4,42 @@ What's new
 See [GitHub releases page](https://github.com/OSOceanAcoustics/echopype/releases) for the complete history.
 
 
+# v0.7.0 (2023 March 24)
+
+## Overview
+
+This release includes new features to interface with Echoview ECS files for computing Sv, reorganization of computing functions into new subpackages, addition of data processing levels to data products from a subset of computing functions, a number of additions and changes associated with metadata, and bug fixes for environmental and calibration parameters intake for computing Sv.
```
Comment (emiliom) [inline docs/source/whats-new.md:11]:
```suggestion
This release includes new features to interface with Echoview ECS files for computing Sv, reorganization of computing functions into new subpackages, addition of data processing level attributes to data products from a subset of computing functions, a number of additions and changes associated with metadata, and bug fixes for environmental and calibration parameters intake for computing Sv.
```

File: docs/source/whats-new.md [hunk 4]
```diff
@@ -4,6 +4,42 @@ What's new
...
+  - `clean`: `remove_noise`, `estimate_noise`
+  - `commongrid`: `compute_MVBS`, `compute_MVBS_index_binning`
+- Add `commongrid.compute_NASC` (#1005)
+  - The current implementation uses brute force looping for mean Sv computation, this will be refactored and optimized together with other functions requiring the same pattern in an upcoming release
+- Add global attributes for data processing levels (#1001)
+  - See **DOC** for functions and conditions under which such attributes are added
+- Standardize sonar metadata for EK80 data (#992)
+    - `sonar_serial_number` is now an empty global attribute, no longer a variable, as in the EK60 case
+    - `transducer_name`, `transducer_serial_number`, `transceiver_serial_number` based on parser parameter `transducer_name`, `transducer_serial_number`, and `serial_number`, respectively
+
```
Comment (emiliom) [inline docs/source/whats-new.md:30]: *(If we merge PR #1010)*, mention the improvements to `apply_mask`, now allowing multi-channel Sv as input.

**Review by emiliom — COMMENTED** [review]
I'm done. Looks great! I made mostly minor suggestions, plus one possible addition (improvements to `apply_mask`, if they make it into the release)

File: docs/source/whats-new.md [hunk 5]
```diff
@@ -4,6 +4,43 @@ What's new
...
+  - Expand ECS parser to accept frequency-dependent values in EK80 ECS files
+- Overhaul `env_params` to ensure correct intake for calibration (#985)
+  - Now allows using `env_params` entries that are xr.DataArrays
+- Move functions previously in `preprocess` subpackage to new subpackages (#993). Calling these functions from `preprocess` is deprecated and will be removed in v0.7.1.
+  - `clean`: `remove_noise`, `estimate_noise`
+  - `commongrid`: `compute_MVBS`, `compute_MVBS_index_binning`
+- Add `commongrid.compute_NASC` (#1005)
+  - The current implementation uses brute force looping for mean Sv computation, this will be refactored and optimized together with other functions requiring the same pattern in an upcoming release
+- Add global attributes for data processing levels (#1001).
+  - This functionality is in a **beta** testing stage.
```
Comment (leewujung) [inline docs/source/whats-new.md:26]:
```suggestion
  - This functionality is in a **beta** testing stage
```

File: docs/source/whats-new.md [hunk 6]
```diff
@@ -4,6 +4,43 @@ What's new
...
+  - See **DOC** for functions and conditions under which such attributes are added
```
Comment (leewujung) [inline docs/source/whats-new.md:27]:
```suggestion
  - See [data processing level specifications](https://echopype.readthedocs.io/en/stable/processing-levels.html) for functions and conditions under which such attributes are added
```

File: docs/source/whats-new.md (hunk 4 above)
Comment (leewujung) [reply docs/source/whats-new.md:32]: Added! I think we're good to go with #1010 once the tests run through.

**Pull request merged**